src/applications/vaos/reducers/appointments.js

```diff
@@ -183,6 +193,82 @@ export default function appointmentsReducer(state = initialState, action) {
         future: null,
         futureStatus: FETCH_STATUS.notStarted,
       };
+    case FETCH_EXPRESS_CARE_WINDOWS:
+      return {
+        ...state,
+        expressCare: {
+          ...initialState.expressCare,
+          fetchWindowsStatus: FETCH_STATUS.loading,
+        },
+      };
+    case FETCH_EXPRESS_CARE_WINDOWS_SUCCEEDED: {
+      const { facilityData } = action;
+      const now = moment.utc();
```

Comment:
To avoid dealing too much with the offsets and local time, all sorting and comparisons are done using UTC.

Reply:
Makes sense, though I think you should pass in the current time through the action, since reducers should always return the same data when the same arguments are passed to it.

Reply:
Okay, passing through the action now.